.github/workflows/unit_test.yml

```diff
@@ -8,6 +8,8 @@ on:
 
 env:
   POETRY_VERSION: "1.6.1"
+  LLAMA_CLOUD_BASE_URL: ${{ secrets.PLATFORM_BASE_URL }}
```

Comment:
do we need this? assuming we are not running integration tests here, seem extra.

Reply:
yea thats stay stuff. We can remove for now